src/shippers/fullstory/src/types.ts

```diff
@@ -6,61 +6,13 @@
  * Side Public License, v 1.
  */
 
+import type { FSApi } from '@fullstory/snippet';
```

Comment:
Hopefully, from now on (+ the [renovate rules](https://github.com/elastic/ebt/pull/3)) we'll know when new versions are released.